app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -96,25 +96,21 @@ fun ViewProfileScreen(
             navigationActions = navigationActions,
             tabList = LIST_TOP_LEVEL_DESTINATION,
             selectedItem = Route.VIEW_PROFILE)
-      },
-      floatingActionButton = {
-        if (userState?.public?.userId != null) {
-          if (userState?.details?.isAdmin == true) {
-            FloatingActionButton(
-                onClick = { navigationActions.navigateTo(Screen.ADMIN) },
-                modifier = Modifier.testTag("AdminFAB"),
-                containerColor = MaterialTheme.colorScheme.primary) {
-                  Text(text = "Admin", color = MaterialTheme.colorScheme.onPrimary)
-                }
-          }
-        }
       }) { innerPadding ->
         Box(Modifier.fillMaxSize().padding(innerPadding)) {
+          if (userViewModel.currentUser.collectAsState().value?.details?.isAdmin == true) {
```

Comment:
Is it me or you swapped the iconbutton of the signout with the one of the adminbutton ?

Reply:
Thanks for noticing!